src/router-hooks.tsx

```diff
@@ -68,14 +68,34 @@ type RouterContextValue = [
 
 const RouterContext = createContext<RouterContextValue | undefined>(undefined);
 
-const parse = (search: string): RouterState => {
+const parse = (pathname: string, search: string): RouterState => {
   const params = new URLSearchParams(search);
-  return {
-    tab: params.get("tab") ?? undefined,
-    api: anchorForParam(params.get("api")),
-    reference: anchorForParam(params.get("reference")),
-    idea: anchorForParam(params.get("idea")),
-  };
+  if (params.get("tab")) {
+    // Legacy
+    return {
+      tab: params.get("tab") ?? undefined,
+      api: anchorForParam(params.get("api")),
+      reference: anchorForParam(params.get("reference")),
+      idea: anchorForParam(params.get("idea")),
+    };
+  }
```

Comment:
We can delete this if we don't use it ourselves.

Reply:
No links here: https://github.com/microbit-foundation/microbit-org-backups/search?q=%22https%3A%2F%2Fpython.microbit.org%2Fv%2F2%22